packages/host/app/components/operator-mode/card-adoption-chain.gts

```diff
@@ -21,7 +21,7 @@ interface Signature {
     file: Ready;
     cardInheritanceChain: CardInheritance[];
     moduleSyntax: ModuleSyntax;
-    openDefinition: (
+    goToDefinition: (
```

Comment:
I renamed this because this function is used anytime navigating to a definition. Not necessarily clicking on an inherited definition